Encountered SSLError: HTTPSConnectionPool(host='api.openai.com', port=443): Max retries exceeded with url: /v1/chat/completions (Caused by SSLError(CertificateError("hostname 'api.openai.com' doesn't match 'nac.int.colorado.edu'")))
Retrying in 5 seconds...
Encountered SSLError: HTTPSConnectionPool(host='api.openai.com', port=443): Max retries exceeded with url: /v1/chat/completions (Caused by SSLError(CertificateError("hostname 'api.openai.com' doesn't match 'nac.int.colorado.edu'")))
Retrying in 5 seconds...
Encountered SSLError: HTTPSConnectionPool(host='api.openai.com', port=443): Max retries exceeded with url: /v1/chat/completions (Caused by SSLError(CertificateError("hostname 'api.openai.com' doesn't match 'nac.int.colorado.edu'")))
Retrying in 5 seconds...
results_str:  {"pattern": "grid", "landmark_offset": 1, "avoid": ["planted area", "conmods"], "seed": True}
